django-verdant/rca/static/rca/js/site.js

```diff
@@ -718,10 +718,45 @@ function onDocumentReady(jQuery, inLightBox){
       return dontShowCookieNotice;
     };
 
+    var displaySurveyBanner = function(context, settings) {
+        // Notice and message
+        $('#survey-banner').addClass('survey-banner--show');
+
+        // Close button
+        $(document).delegate('#survey-banner-close', 'click', function() {
+          $(".survey-banner").slideUp("slow");
+          return false;
+        });
+
+        $(".survey-banner").slideDown("slow");
+
+        // Set notice to never show again by default.
+        dontShowSurveyBannerAgain();
+      };
+
+      var dontShowSurveyBannerAgain = function() {
+        // domainroot root variable is set in base.html - needed so the same cookie works across all subdomains
+
+        // set or extend the cookie life for a year
+        var exdate = new Date();
+        exdate.setDate(exdate.getDate() + 365);
+        document.cookie = "dontShowSurveyBanner=" + "TRUE; expires=" + exdate.toUTCString() + ";domain=" + domainroot + ";path=/";
+      };
+
+      var pleaseShowSurveyBanner = function() {
+        // Don't show the notice if we have previously set a cookie to hide it
+        var dontShowSurveyBanner = (document.cookie.indexOf("dontShowSurveyBanner") != -1) ? false : true;
+        return dontShowSurveyBanner;
```

Comment:
I think this could be shortened to:

```suggestion
        return document.cookie.indexOf("dontShowSurveyBanner") !== -1;
```